lib/src/widgets/chat_groupedlist_widget.dart

```diff
@@ -210,23 +205,19 @@ class _ChatGroupedListWidgetState extends State<ChatGroupedListWidget>
                   )
                 : _chatStreamBuilder,
           ),
-          widget.showTypingIndicator
-              ? TypingIndicator(
-                  typeIndicatorConfig: widget.typeIndicatorConfig,
-                  chatBubbleConfig: chatBubbleConfig?.inComingChatBubbleConfig,
-                  showIndicator: widget.showTypingIndicator,
-                )
-              : ValueListenableBuilder(
-                  valueListenable: ChatViewInheritedWidget.of(context)!
-                      .chatController
-                      .typingIndicatorNotifier,
-                  builder: (context, value, child) => TypingIndicator(
+          ValueListenableBuilder(
+            valueListenable: ChatViewInheritedWidget.of(context)!
+                .chatController
+                .typingIndicatorNotifier,
+            builder: (context, value, child) => value
+                ? TypingIndicator(
                     typeIndicatorConfig: widget.typeIndicatorConfig,
                     chatBubbleConfig:
                         chatBubbleConfig?.inComingChatBubbleConfig,
                     showIndicator: value,
-                  ),
-                ),
+                  )
+                : Container(),
```

Comment:
You should not do this. As when we pass `false` to `TypingIndicator`'s `showIndicator` field then it animates back in which is lost if we just show a Container on `false`. Can you please remove this?

Reply:
Good catch, is removed. I misunderstood the previous if statement.